lib/reports/published_attachments_report.rb

```diff
@@ -0,0 +1,40 @@
+module Reports
+  class PublishedAttachmentsReport
+    def report
+      path = Rails.root.join("tmp/attachments_#{Time.zone.now.strftime('%d-%m-%Y_%H-%M')}.csv")
+      csv_headers = ["Organisation", "Filename", "Filetype", "Published Date"]
+
+      attachments = Attachment.find_by_sql([
```

Comment:
We can't do this query using ActiveRecord as the relation between Attachment and Edition (via. `attachable`) is polymorphic, so such as query would involve multiple steps which exceeded the machine's memory.